integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -231,6 +232,34 @@ public void startKerberosHiveContainer() {
     }
   }
 
+  public void startJdbcHiveContainer(Map<String, String> envVars) {
```

Comment:
I think it better to change `startJdbcHiveContainer()` to `startSqlBaseAuthHiveContiner()`?

Reply:
Renamed.